package.json

```diff
@@ -9,10 +9,16 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/webpacker": "5.2.1",
     "arctic_admin": "^3.2.0",
+    "eslint-config-standard": "^16.0.2",
```

Comment:
why is eslint a prod dependency and not a dev dependency?

Reply:
My bad, I will change it 👌